IFC4x3/PropertyConstants/r/RIGID_2bdxKsu9b0HvmMMw43kior.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RIGID_2bdxKsu9b0HvmMMw43kior" Name="RIGID" UniqueId="a59fb536-e099-4047-9c16-5ba103baccb5" />
```

Comment:
The above and this are the same? Something smells fishy. This should not happen.